students/caitlin/lecture_notes/javascript_1.js

```diff
@@ -2,24 +2,26 @@
 //Numbers
 
 // 1. Declare a variable called age and give it a value of your age.
+var age = 33;
 
 // 2. Declare a variable called showMeSix and give it a value of 3 + 3.  We can perform math operations on numbers.
-
+var showMeSix = 3 + 3;
 // Strings
 // 3. Declare a variable called name and give it a value of your name.
-
+var name = 'Caitlin';
 // 4. Declare a variable called greeting and give it a value of a nice greeting.  Use either template strings or concatenation to add your name to the end of the greeting.
-
+var greeting = "Welcome" + ${name};
 // Booleans
 // 5. Declare a variable called lovesCode and give it a value of a boolean.
-
+var lovesCode = true;
 // Arrays
 // 6. Declare a variable called friends and give it a value of an array containing 3 strings, each one a name of a friend.
 //Console.log the first item in your array
-
+var friends = ['Alex', 'Winter', 'Sinead'];
+console.log(0)
```

Comment:
```suggestion
console.log(friends[0])
```
Your code would just bring the number zero. We want a specific index in the array. Try adding different numbers as the index to see what happens